guides/v2.2/frontend-dev-guide/themes/theme-create.md

```diff
@@ -260,7 +260,10 @@ To learn more about theme layouts, refer to the [Layout section]({{page.baseurl}
 
 ## Troubleshooting {#trouble}
 
-When your theme changes are not visible even after clearing the cache, try redeploying your static files using `magento setup:static-content:deploy en_US` (replacing the language/locale as appropriate).
+When your theme changes are not visible even after clearing the cache, try redeploying your static files using `magento setup:static-content:deploy` or add the `-f` argument to force deploy static content in any deployment mode in case you aren't in production mode. 
```

Comment:
```suggestion
When your theme changes are not visible even after clearing the cache, try redeploying your static files using the `magento setup:static-content:deploy` command, or add the `-f` argument to force deploy static content in any deployment mode in case you are not in production mode. 
```